<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_form</class>
 <widget class="QDialog" name="add_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>اضافه كلمات </string>
  </property>
  <widget class="QPushButton" name="add_b">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>اضافة</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_b">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>رجـــــــوع</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="en_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ar_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>الكلمة الانجليزية</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>الكلمة العربية</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
